src/wasi/Wasi.h

```diff
@@ -112,22 +115,115 @@ class WASI {
     } wasi_errno_t;
 #undef TO_ENUM
 
+    typedef uint32_t wasi_size_t;
+
+    typedef uint32_t wasi_fd_t;
+
+    typedef enum wasi_oflags : uint16_t {
+        oflag_create,
+        oflag_directory,
+        oflag_excl,
+        oflag_trunc,
+    } wasi_oflags_t;
+
+    typedef enum wasi_lookupflags : uint32_t {
+        symlink_follow,
+    } wasi_lookupflags_t;
+
+    typedef enum wasi_rights : uint64_t {
```

Comment:
These enum values will be used later?

Reply:
They were used with an earlier version, though it may not be needed now that we use uvwasi, so I wil remove them for the time being and add them if needed again.